Sources/TuistGenerator/Generator/ConfigGenerator.swift

```diff
@@ -193,8 +193,7 @@ final class ConfigGenerator: ConfigGenerating {
 
         // Info.plist
         if let infoPlist = target.infoPlist, let path = infoPlist.path {
-            let relativePath = path.relative(to: sourceRootPath).pathString
-            settings["INFOPLIST_FILE"] = .string("\(relativePath)")
+            settings["INFOPLIST_FILE"] = .string("\(path.pathString)")
         }
```

Comment:
Ouch this will cause a regression :/ 

The generated projects can't contain absolute paths as it will cause unstable project generation based on generation location / different machines etc... (i.e. one would no longer be able to check in a generated project).

Which issue is this trying to solve? We may be able to find an alternative.

Reply:
Huh, yeah, I did not think about checking in a generated project.

The problem is that the `.xcodeproj` can be now generated into two places (and therefore the one generated into to `~/.tuist/Cache` would not find the infoplist file). But I think that leveraging `$(SRCROOT)` should do the trick here?

Reply:
> But I think that leveraging `$(SRCROOT)` should do the trick here?

`$SRCROOT` will by default still point to the XcodeProj parent directory, to get the desired path I believe the mapper would need to set the `project.sourceRootPath = projectsDirectory` to allow all paths to work correctly _(e.g. a few other elements rely on relative paths to `sourceRootPath`, including entitlements, framework & header search paths)_

Slightly related: Using `$SRCROOT` is technically correct too in general, I was curious why it wasn't used in the first place, turns it out it was explicitly removed to fix a fastlane issue in the past :/ https://github.com/tuist/tuist/pull/511 explicitly removed